ui/widgets/calendar.js

```diff
@@ -45,8 +45,18 @@ return $.widget( "ui.calendar", {
 	version: "@VERSION",
 	options: {
 		buttons: [],
+		classes: {
```

Comment:
This is fine if these are the only places we have style classes. We just need to document the full list of structural classes in the API docs.